packages/common/src/types/search.ts

```diff
@@ -3,17 +3,31 @@ import { Table } from './explore';
 import { Dimension, Metric } from './field';
 import { ChartKind, SavedChart } from './savedCharts';
 import { Space } from './space';
+import {
+    ValidationErrorChartResponse,
+    ValidationErrorDashboardResponse,
+} from './validation';
 
 export type SpaceSearchResult = Pick<Space, 'uuid' | 'name' | 'uuid'>;
 export type DashboardSearchResult = Pick<
     Dashboard,
     'uuid' | 'name' | 'description' | 'spaceUuid'
->;
+> & {
+    validationErrors: Pick<
+        ValidationErrorDashboardResponse,
+        'error' | 'createdAt' | 'validationId'
```

Comment:
`createdAt` isn't used in the rendering, will it be?

Reply:
Yea, good point, I'm only keeping `validationId`s for now then 👍